packages/atomic/src/components/common/facets/facet-tabs/facet-tabs-utils.ts

```diff
@@ -0,0 +1,39 @@
+import {CategoryFacet, DateFacet, Facet, NumericFacet} from '@coveo/headless';
+import {shouldDisplayOnCurrentTab} from '../../../../utils/tab-utils';
+import {AnyBindings} from '../../interface/bindings';
+
+type AnyFacetType = Facet | NumericFacet | CategoryFacet | DateFacet;
+
+/**
+ * Updates the visibility of a facet based on the active tab.
+ * @param tabs - The active tab.
+ * @param facet - The facet to update.
+ * @param bindings - The bindings object.
+ * @returns A boolean indicating whether the facet was enabled or disabled.
+ */
+export function updateFacetVisibilityForActiveTab(
+  tabs: string,
```

Comment:
The doc is indeed wrong! 

I can do that, but my thinking was that since the logic is the same on all the facets, it would be simpler (and make it easier to modify with the upcoming facet refactoring) to use a shared function.